source/_integrations/swiss_public_transport.markdown

```diff
@@ -15,9 +15,9 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The Swiss public transport integration will give you the next three departure times from a given location to another one in Switzerland.
+The `swiss_public_transport` sensor will give you the next three departure times from a given location to another one in Switzerland.
```

Comment:
```suggestion
The Swiss public transport integration will give you the next three departure times from a given location to another one in Switzerland.
```
Here, we spell the integration the way the user sees it in the UI.
![image](https://github.com/home-assistant/home-assistant.io/assets/38767475/bbae600c-30fd-46b4-8068-d67a6f482ea6)